app-modules/form/src/Filament/Blocks/SelectFormFieldBlock.php

```diff
@@ -63,6 +63,9 @@ public function fields(): array
         ];
     }
 
+    /**
+     * @return array<string, string>
```

Comment:
This return type is not fully correct as some of these values can be something other than `string` like `null` for example.